### DISCUSSION TITLE
Quarkus 3.5.1 released - Maintenance release

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#36993] [Quarkus Blog/Website]
# Quarkus 3.5.1 released - Maintenance release

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-3-5-1-released/

**Comment by bulldog98** [comment]
The CVE link does show an empty page without details, I guess it's to give people a few weeks to upgrade their applications?

**Reply by gsmet** [reply]
It's supposed to have been unembargoed but apparently it takes more time than expected :).
